components/css/global/_base.scss

```diff
@@ -74,28 +83,28 @@ h1,h2,h3,h4,h5,h6{
 		font-weight:inherit;
 	}
 }
-h1,h2,h3,h4,h5,h6{
-	margin-top:0;
-	margin-bottom:0;
-}
-h1+h3{
-	padding-top:0;
-}
-h3+h6,h4+h6{
-	padding-top:1em;
-}
-h4+h5,h5+h6{
-	padding-top:0.75em;
-}
-h2+h6,h3+h5,h3+h4{
-	padding-top:0.5em;
-}
-h1+h4,h1+h5,h1+h6,h2+h3,h2+h4,h2+h5{
-	padding-top:0.25em;
-}
-h1+h2{
-	padding-top:0.15em;
-}
+// h1,h2,h3,h4,h5,h6{
+// 	margin-top:0;
+// 	margin-bottom:0;
+// }
+// h1+h3{
+// 	padding-top:0;
+// }
+// h3+h6,h4+h6{
+// 	padding-top:1em;
+// }
+// h4+h5,h5+h6{
+// 	padding-top:0.75em;
+// }
+// h2+h6,h3+h5,h3+h4{
+// 	padding-top:0.5em;
+// }
+// h1+h4,h1+h5,h1+h6,h2+h3,h2+h4,h2+h5{
+// 	padding-top:0.25em;
+// }
+// h1+h2{
+// 	padding-top:0.15em;
+// }

```

Comment:
Are these necessary anymore since the headers now default to the largest margin to determine the distance between them?

Reply:
Gone.